rpcs3/Emu/Cell/lv2/sys_mmapper.cpp

```diff
@@ -629,13 +629,17 @@ error_code sys_mmapper_map_shared_memory(ppu_thread& ppu, u32 addr, u32 mem_id,
 			return CELL_EALIGN;
 		}
 
-		while (!mem.shm)
+		for (stx::shared_ptr<std::shared_ptr<utils::shm>> to_insert, null; !mem.shm;)
```

Comment:
what is null for ?

Reply:
its to compare with old value, and save the old value in case of commit failure.